lib/Basics/files.cpp

```diff
@@ -1150,7 +1150,6 @@ int TRI_VerifyLockFile(char const* filename) {
     LOG_TOPIC(WARN, arangodb::Logger::FIXME) << "found existing lockfile '" << filename << "' of previous process with pid " << pid << ", and that process seems to be still running";
   }
 
-#ifdef TRI_HAVE_SETLK
```

Comment:
is `F_SETLK` available on windows ?

Reply:
I think you should probably rather add the define for MacOs in `lib/Basics/operating-system.h:` alternatively remove it completely